presto-hive/src/main/java/io/prestosql/plugin/hive/RetryDriver.java

```diff
@@ -31,10 +31,10 @@
 public class RetryDriver
 {
     private static final Logger log = Logger.get(RetryDriver.class);
-    private static final int DEFAULT_RETRY_ATTEMPTS = 10;
-    private static final Duration DEFAULT_SLEEP_TIME = Duration.valueOf("1s");
-    private static final Duration DEFAULT_MAX_RETRY_TIME = Duration.valueOf("30s");
-    private static final double DEFAULT_SCALE_FACTOR = 2.0;
+    public static final int DEFAULT_RETRY_ATTEMPTS = 10;
+    public static final Duration DEFAULT_SLEEP_TIME = Duration.valueOf("1s");
```

Comment:
While we're changing these, please update these calls to use the non-parsing versions (doesn't really matter but is cleaner and matches other places):
```java
public static final Duration DEFAULT_SLEEP_TIME = new Duration(1, SECONDS);
public static final Duration DEFAULT_MAX_RETRY_TIME = new Duration(30, SECONDS);
```

Reply:
ok